packages/protobuf-test/src/perf.ts

```diff
@@ -95,173 +102,163 @@ interface Test {
 }
 
 function setupTests(): Test[] {
-  const tests: Test[] = [];
```

Comment:
This looks like a huge change but I'm just switching to generating the benchmark cases instead of having to repeat code.